### PR TITLE
chore: add social network example

### PR DESCRIPTION
**Pull request by fmvilas**
**Description**

This example was created as part of the [talk Fran Mendez did at QCon Plus 2021](https://plus.qconferences.com/plus2021/speakers/fran-mendez). I thought it would be cool to have it here too.

### CODE REVIEW

File: examples/social-network/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Social Network Example
+
+This example was created as part of the [talk Fran Mendez did at QCon Plus 2021](https://plus.qconferences.com/plus2021/speakers/fran-mendez).
+
+## Run it
+
+There are three services. All of them can be started by running:
```
Comment (smoya) [inline examples/social-network/README.md:7]: Do you have by any chance a diagram of how the apps are interconnected? I think it is worth adding more context here.

Comment (fmvilas) [reply examples/social-network/README.md:11]: I didn't have it but I created it just for you right now 😘 :trollface:

Comment (boyney123) [reply examples/social-network/README.md:11]: Thanks for the diagram, helped me understand what's going on.

Small one @fmvilas you have 

- server folder

Do you think it's worth naming that `websocket-server` to match the diagram?

Comment (fmvilas) [reply examples/social-network/README.md:11]: 👍 Done!

**Comment by fmvilas** [comment]
> Think the Example is great @fmvilas, I wonder if your README file if we could add a `How it works` section?
> 
> I understand how Glee works but I had to dig around the code to get an understanding of how it exactly works, for example which part of this application is Glee helping with? It might be useful to explain to people how glee was used in the example?
> 
> What do you think?

Yeah, not in this PR but opened this follow up issue: https://github.com/asyncapi/glee/issues/206. Thanks, Dave!

File: examples/social-network/frontend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```
Comment (jonaslagoni) [inline examples/social-network/frontend/.gitignore:6]: 🤣 First I thought it said `php`, and I was like 🤨

File: examples/social-network/notifications-service/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "glee-hello-world",
+  "version": "0.1.0",
+  "description": "Hello world app to improve Glee onboarding ",
```
Comment (jonaslagoni) [inline examples/social-network/notifications-service/package.json:4]: 🧐

File: examples/social-network/websocket-server/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "glee-hello-world",
+  "version": "0.1.0",
+  "description": "Hello world app to improve Glee onboarding ",
```
Comment (jonaslagoni) [inline examples/social-network/websocket-server/package.json:4]: 🧐 Should have been websocket-server details right?

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Just 2 comments 🙂

File: examples/social-network/notifications-service/package.json (hunk 3 above)
Comment (fmvilas) [reply examples/social-network/notifications-service/package.json:4]:
```suggestion
  "name": "glee-hello-world",
  "version": "0.1.0",
```

File: examples/social-network/websocket-server/package.json (hunk 4 above)
Comment (fmvilas) [reply examples/social-network/websocket-server/package.json:4]:
```suggestion
  "name": "glee-hello-world",
  "version": "0.1.0",
```

**Comment by fmvilas** [comment]
Changes applied 😄 I don't care about the description of these apps, they're just examples so I better remove them haha!

PS: Love the auto-review hack with suggestions so I don't have to check it out on my machine 😂

**Review by jonaslagoni — APPROVED** [review]
👍

**Pull request merged**